src/base/collection.js

```diff
@@ -749,18 +750,78 @@ CollectionBase.prototype._reset = function() {
 // Lodash methods that we want to implement on the Collection.
 // 90% of the core usefulness of Backbone Collections is actually implemented
 // right here:
-const methods = ['forEach', 'each', 'map', 'collect', 'reduce', 'foldl',
-  'inject', 'reduceRight', 'foldr', 'find', 'detect', 'filter', 'select',
-  'reject', 'every', 'all', 'some', 'any', 'include', 'contains', 'invoke',
-  'max', 'min', 'toArray', 'size', 'first', 'head', 'take', 'initial', 'rest',
-  'tail', 'drop', 'last', 'without', 'difference', 'indexOf', 'shuffle',
-  'lastIndexOf', 'isEmpty', 'chain'];
+
+const methods = {
```

Comment:
Yeah, tbh I think we should drop the lodash methods entirely. But that's probably even more annoying.

Would you be able to note any changed methods in the [`changelog`](https://github.com/tgriesser/bookshelf/blob/master/CHANGELOG.md) please?